.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
  - "node"
 env:
+ - PLATFORM_SERVICE=core/jazz_acl
```

Comment:
Add the authorizers to this list if unit tests are aded.

Reply:
authorizer unit test needs more work